src/mock/utils/fnr-utils.ts

```diff
@@ -51,4 +60,9 @@ function getFiresifretÅr(fødselsnummer: string) {
         return year + 1900;
     }
 
+}
+
+function getRiktigKjønnPåFødslesnummer(fødselsnummer: string, kjønn: Kjønn) {
+    const kjønnsiffer = kjønn === Kjønn.Kvinne ? '4' : '5';
+    return fødselsnummer.substr(0, 8) + kjønnsiffer + fødselsnummer.substr(9, fødselsnummer.length);
```

Comment:
Hvis du manipulerer slik på fødselsnummeret, vil det da fremdeles være gyldig? 🤔

Reply:
Når man søker opp fødselsnummere i topp-menyen, er det endel sjekker som sjekker at det er gyldig.

Reply:
Ja... Du sier noe. Det vil det nok ikke, men jeg har bare gjort dette nå for far/mor, som ikke er videre søkbart slik det er i dag. Dette er mocking og disse fødselsnummerne vil vel ikke uten videre kunne søkes opp uansett. Så er det et problem?

Reply:
er det eventuellt en mulighet å genrere fødselsnummere til du får et med riktig kjønn?

Reply:
Eller så kan du bare endre de to siste tallene (kontrollsiffrene) slik det stemmer igjen etter at du har endret kjønn-sifferet :)

> Hvert av kontrollsifrene er en sjekksum beregnet ut fra de foregående sifrene. Hvert av sifrene  foran multipliseres med hvert sitt tall, og resultatene legges sammen og deles på 11. Kontrollsifferet er da 11 minus resten av den siste divisjonen, med ett unntak: dersom resten er 0 og man ender opp med 11 − 0 er kontrollsifferet 0, og ikke 11. Hvis de 9 første sifrene av et fødselsnummer er d1 d2 m1 m2 å1 å2 i1 i2 i3, er første kontrollsiffer:

> k1 = 11 - ((3 × d1 + 7 × d2 + 6 × m1 + 1 × m2 + 8 × å1 + 9 × å2 + 4 × i1 + 5 × i2 + 2 × i3) mod 11),
> der mod er modulusfunksjonen. Tilsvarende er det andre kontrollsifferet:

> k2 = 11 - ((5 × d1 + 4 × d2 + 3 × m1 + 2 × m2 + 7 × å1 + 6 × å2 + 5 × i1 + 4 × i2 + 3 × i3 + 2 × k1) mod 11).

Her kan du kose deg med litt matte også 😀

Reply:
God idé :) Da gjør jeg det.